### DISCUSSION TITLE
[New Feature] Hotkey for requesting pick up by Carryall

### DISCUSSION THREAD

**Discussion by TAK02** [Phobos-developers/Phobos#947] [Ideas and feedback]
### Description:
Emperor: Battle for Dune, not sure about Dune 2000 or Dune 2, had a hotkey that calls a `Carryall=yes` unit to pick up the selected unit.
In EBfD that feature was restricted to the "Advanced" Carryalls to separate them from the Harvester Carryalls.

Not sure how EBfD handled multiple units selected and Request Carryall key pressed, but it'd be nice if all Carryalls would split the selected units amongst themselves for pick up.

**Comment by secsome** [comment]
Carryall is restored by Ares IIRC? Maybe this should be Alex's job to implement it.

**Comment by amr-el-sayed** [comment]
Similarly the repair button over any ground vehicle can call the carryall to lift it to the repair depot. Would be a great enhancement!
